### DISCUSSION TITLE
multi-label segmentation issues

### DISCUSSION THREAD

**Discussion by shenao1995** [Project-MONAI/MONAI#6776] [General]
I'm attempting a multi-label segmentation task for the spine. But I have three cases: one label corresponds to one vertebra, two masks correspond to two vertebrae, and three masks correspond to three vertebrae. In general, the segmentation network outputs three channels representing three-classes segmentation tasks. But when I encounter data with only one mask or two masks, I will definitely report an error. I would like to ask how to solve this situation.

**Comment by KumoLiu** [comment]
Hi, @shenao1994 If I understand correctly, your label may have 4 values: 0, the background, 1, 2, and 3, the first, second, and third vertebrae, respectively. You must then one-hot your label. See [this tutorial] (https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb) for more information.
Hope it helps, thanks!

**Reply by shenao1995** [reply]
Okay, thank you very much for your help！
